### ISSUE TITLE
`describe select *` doesn't return results in python interface

### ISSUE THREAD

**Issue by david-cortes** [duckdb/duckdb#5799]
### What happens?

The docs suggest usage of `describe select *` for getting info about column names and types of parquet files:
https://duckdb.org/docs/data/parquet
> If you want to figure out the column names and types contained within a Parquet file it is easier to use DESCRIBE.
```sql
-- fetch the column names and column types
DESCRIBE SELECT * FROM 'test.parquet';
```

But this doesn't return anything when executed as a query, neither with parquet files nor with other input types.

In the latest pre-release version, executing the query prints something (looks like a black rectangle when executed in python), but the result is not returned as a table.

### To Reproduce

```python
import numpy as np, pandas as pd
df = pd.DataFrame({"col1" : np.arange(5)})
import duckdb
duckdb.query("describe select * from df").to_df()
```

### OS:

Linux

### DuckDB Version:

0.6.1

### DuckDB Client:

Python

### Full Name:

David Cortes

### Affiliation:

None

### Have you tried this on the latest `master` branch?

- [X] I agree

### Have you tried the steps to reproduce? Do they include all relevant data and configuration? Does the issue you report still appear there?

- [X] I agree

**Comment by Alex-Monahan** [comment]
Hello! This works for me in the CLI shell, so it is possible that it is a feature only in the CLI. Does this query work for you?
```sql
describe select * from range(10) t(x);
```

**Comment by david-cortes** [comment]
Actually, I'm suspecting this is a python-specific bug, as the query would correctly return a table as result in R:
```r
library(DBI)
library(duckdb)
conn = dbConnect(duckdb::duckdb(), dbdir=":memory:", read_only=FALSE)
dbGetQuery(conn, "describe select * from range(10) t(x);")
```

**Comment by Tishj** [comment]
I can confirm that this is a bug.

I'm not sure how you're getting a weird black rectangle though, when I run this the `query` returns None, and the `.to_df()` fails because of that

**Comment by Mytherin** [comment]
> I can confirm that this is a bug.
> 
> I'm not sure how you're getting a weird black rectangle though, when I run this the `query` returns None, and the `.to_df()` fails because of that

The query seems to always print the progress bar for me even though it completes instantly - perhaps that is where the black rectangle comes from?

**Comment by david-cortes** [comment]
This is now solved in 0.7.0.
